src/app/components/ebi-submission-menu/ebi-submission-menu.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'ebi-submission-menu',
+  templateUrl: './ebi-submission-menu.component.html',
+  styleUrls: ['./ebi-submission-menu.component.scss'],
+})
+export class EbiSubmissionMenuComponent implements OnInit {
+  @Input() submission: Object;
```

Comment:
It would be better to define an interface than a generic Object.

Reply:
As we agreed. this is gonna be a bit complex. i've created a new ticket for cleaning up all the application and using models for all the entities

https://trello.com/c/y3mMjMnS/47-clean-up-using-interfaces-or-models-for-different-entities